syntax = "proto3";
package tinkoff.proto;

// Request
message CreateBeneficiaryBankDetailsRequest {
  string beneficiary_id = 1;
  string data = 2;
}
